Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -35,6 +35,16 @@
 #include <cstdlib>
 #include <random>
 
+#include <regex>
+
+#define KERNEL_NAME_MIINST_REGEX "_MI(\\d+)x(\\d+)x(\\d+)x(\\d+)_"
+#define KERNEL_NAME_MIINST_REGEX_NUM_MATCHES \
+    5 //1 (index 0) for entire match and 4 for the sub-experssion matches
+
+#define KERNEL_NAME_GSUA_REGEX "_GSUA([S|M])B_"
+#define KERNEL_NAME_GSUA_REGEX_NUM_MATCHES \
+    2 //1 (index 0) for entire match and 1 for the sub-experssion matches
+
```

Comment:
We only used these definitions once in the function. Can we not define them and just use the actual definition in the function?